packages/terra-site/src/examples/dynamic-grid/Dashboard.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import DynamicGrid from 'terra-dynamic-grid/src/DynamicGrid';
+import Region from 'terra-dynamic-grid/src/Region';
+
+import Card from './Card';
+
+const layouts = [{
+  'grid-template-columns': '300px 1fr 300px',
+  'grid-template-rows': 'auto auto auto auto',
+  regions: [
+    {
+      name: 'leftGutter',
+      'grid-column-start': 1,
+      'grid-row-start': 1,
+      'grid-row-end': 5,
+    },
+    {
+      name: 'notification',
+      'grid-column-start': 2,
+      'grid-column-end': 3,
+      'grid-row-start': 1,
+    },
+    {
+      name: 'rightGutter',
+      'grid-column-start': 3,
+      'grid-row-start': 1,
+      'grid-row-end': 5,
+    },
+    {
+      name: 'r1',
+      'grid-column-start': 2,
+      'grid-row-start': 2,
+      style: {
+        padding: '0 5px 5px 5px',
+      },
+    },
+    {
+      name: 'r2',
+      'grid-column-start': 2,
+      'grid-row-start': 3,
+      style: {
+        padding: '5px',
+      },
+    },
+    {
+      name: 'r3',
+      'grid-column-start': 2,
+      'grid-row-start': 4,
+      style: {
+        padding: '5px 5px 0',
+      },
+    },
+  ],
+},
+];
+
+const DashboardLayout = () => (
+  (<DynamicGrid layouts={layouts}>
```

Comment:
Generally I'd prefer to not pass in a monolithic config info the top component, but rather build out each child with the config it needs. However, this was needed to support responsive layouts that change based on media query. I don't have a doc example of this yet, but it would be challenging to pass in the responsive config for each region separate (as its dependent on the grid config) so I determined it made the most sense to just pass in one large object.